custom/enikshay/tests/utils.py

```diff
@@ -141,6 +141,56 @@ def get_adherence_case_structure(case_id, indexed_episode_id, adherence_date, ex
     )
 
 
+def get_prescription_case_structure(case_id, indexed_episode_id, extra_update=None):
+    extra_update = extra_update or {}
+    update = {
+        'state': 'fulfilled',
+    }
+    update.update(extra_update)
+    return CaseStructure(
+        case_id=case_id,
+        attrs={
+            "case_type": "prescription",
+            "create": True,
+            "update": update
+        },
+        indices=[CaseIndex(
+            CaseStructure(case_id=indexed_episode_id, attrs={"create": False}),
+            identifier='episode_of_prescription',
+            relationship=CASE_INDEX_EXTENSION,
+            related_type='episode',
+        )],
+        walk_related=False,  # TODO I'm not sure what this should be
```

Comment:
if this is `True`, when you create a prescription with this `CaseStructure`, it would also create the `episode`, `occurrence`, `person`, etc. 
I think this is fine as is.